applications/NXlauetof.nxdl.xml

```diff
@@ -27,8 +27,22 @@
   xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
   xsi:schemaLocation="http://definition.nexusformat.org/nxdl/3.1 ../nxdl.xsd"
   >
+  <symbols>
+      <doc>
+        The symbol(s) listed here will be used below to coordinate datasets with the same shape.
+      </doc>
+      <symbol name="nXPixels">
+        <doc>Number of x pixels</doc>
```

Comment:
Should x, y, etc all be upper case (here and in other definitions)? There's inconsistent usage.

Reply:
Which way did this get resolved?